TidepoolKit/Model/Prescription/TPrescription.swift

```diff
@@ -0,0 +1,232 @@
+//
+//  TPrescription.swift
+//  TidepoolKit
+//
+//  Created by Darin Krauss on 4/22/21.
+//  Copyright © 2021 Tidepool Project. All rights reserved.
+//
+
+public struct TPrescription: Codable, Equatable {
```

Comment:
FWIW I don't know the official data model, so this review doesn't include my checking on that.  If you want me to re-review it with that in mind, LMK.